crane.hcl

```diff
@@ -0,0 +1,26 @@
+description = "crane - Command line tool for interacting with remote images and registries."
+binaries = ["crane"]
+test = "crane version"
+
+platform "Darwin" "x86_64" {
+  source = "https://github.com/google/go-containerregistry/releases/download/v${version}/go-containerregistry_${os}_${arch}.tar.gz"
+}
+
+platform "Darwin" "arm64" {
+  source = "https://github.com/google/go-containerregistry/releases/download/v${version}/go-containerregistry_${os}_${arch}.tar.gz"
+}
+
+platform "Linux" "x86_64" {
```

Comment:
Unfortunately this won't work as the CPU architecture key that Hermit supports is `amd64`, not `x86_64`.

You'll have to match on `amd64` here (and in the other x86_64 block), and replace `${arch}` in the URL with `x86_64`. eg.

```hcl
platform linux amd64 {
  source = source = "https://github.com/google/go-containerregistry/releases/download/v${version}/go-containerregistry_${os}_x86_64.tar.gz"
}
```

Reply:
Done